src/SmartComponents/Advisor/Advisor.js

```diff
@@ -56,7 +56,7 @@ const Advisor = () => {
 
     const urlRest = `&reports_shown=true&impacting=true&offset=0&limit=10${selectedTags?.length ?
         `&tags=${selectedTags?.join()}` : ''}${workloads?.SAP ? '&sap_system=true' : ''}${SID?.length ? `&sap_sids=${SID?.join()}` : ''}`;
-    const totalRiskUrl = risk => `${UI_BASE}/advisor/recommendations?total_risk=${risk}${urlRest}`;
+    const totalRiskUrl = risk => `${UI_BASE}/advisor/recommendations?sort=total_risk=${risk}${urlRest}`;
```

Comment:
Perhaps you meant
```suggestion
    const totalRiskUrl = risk => `${UI_BASE}/advisor/recommendations?sort=total_risk&total_risk=${risk}${urlRest}`;
```
?

Reply:
And the sort should be descending, thus you have to use `-total_risk`

Reply:
True, updated the PR